src/clr-core/badge/badge.element.ts

```diff
@@ -41,7 +41,9 @@ export class CwcBadge extends LitElement {
 
   render() {
     return html`
-      <slot></slot>
+      <div class="private-host">
```

Comment:
Why `.private-host`?

Reply:
I did not know what to call this (maybe just `.badge`) but its the reasoning described in the PR. Basically we dont want styles directly on host otherwise they are exposed and can be overridden externally without using the css props. Just makes the styles more encapsulated/strict.

Reply:
got it. Thanks!